explainer.md

```diff
@@ -0,0 +1,793 @@
+
+#### OffsetDateTime.prototype.microsecond: number
+
+The `.microsecond` property represents the microseconds of the milliseconds of the `OffsetDateTime `  according to the proleptic Gregorian calendar with a midnight to midnight 24 hour day.
+
+#### OffsetDateTime.prototype.nanosecond: number
+
+The `.nanosecond` property represents the nanoseconds of the microseconds of the `OffsetDateTime `  according to the proleptic Gregorian calendar with a midnight to midnight 24 hour day.
+
+#### OffsetDateTime.prototype.dayOfWeek: number
+
+The `.dayOfWeek` property represents the day of the week according to the proleptic Gregorian calendar with a midnight to midnight 24 hour day where Monday is `1` and `Sunday` is `7` in accordance with ISO-8601. 
+
+#### OffsetDateTime.prototype.dayOfYear: number
+
+The `.dayOfYear` property represents the ordinal day of the Gregorian year according to ISO-8601.
+
+#### OffsetDateTime.prototype.weekOfYear: number
+
+The `.weekOfYear` property represents the ISO week-number. Beware that dates at the begining of a year may be part of a week from the preceding year, and dates at the end of a year may be part of a week at the beginning of the next year, as the first week of any year is defined as the week that contains the first Thursday of the week.
+
+#### OffsetDateTime.prototype.plus(data: IntervalLike): ZonedDateTime
+
+Creates a new `ZonedDateTime` object by adding (subtracting for negative values) values to its members. The specified values must be numeric if specified.
+
+The algorithm is such that:
+
+1. the individual values are added to the existing values.
+2. the range of `nanoseconds` is ensured to be between 0 and 999 by adjusting the `microseconds`
+3. the range of `microseconds` is ensured to be between 0 and 999 by adjusting the `milliseconds`
+4. the range of `milliseconds` is ensured to be between 0 and 999 by adjusting the `seconds`
+5. the range of `seconds` is ensured to be between 0 and 59 by adjusting the `minutes`
+6. the range of `minutes` is ensured to be between 0 and 59 by adjusting the `hours`
+7. the range of `hours` is ensured to be between 0 and 23 by adjusting `days`
+8. the range of `days` is ensured to be between 1 and 29-31 depending on the month by adjusting `month`
+9. the range of `months` is ensured to be between 1 and 12 by adjusting the `years`.
+
+#### OffsetDateTime.prototype.minus(other: ZonedDateTime): Interval
+#### OffsetDateTime.prototype.withZone(iana: string) : ZonedDateTime
+#### OffsetDateTime.prototype.toString() : string
+
+This creates an ISO-8601 string in the following format **`year`-`month`-`day`T`hours`:`minutes`:`seconds`.`nanoseconds``offset`**.
+
+The `year` is 0-padded to a minimum of 4 digits. `month`, `day`, `hours`, `minutes`, `seconds` are 0-padded to a minimum of 2 digits. `nanoseconds` is 0-padded to a minimum of 9 digits. The `offset` is the timezone offset as created by `zoned.offset.
```

Comment:
```suggestion
The `year` is 0-padded to a minimum of 4 digits. `month`, `day`, `hours`, `minutes`, `seconds` are 0-padded to a minimum of 2 digits. `nanoseconds` is 0-padded to a minimum of 9 digits. The `offset` is the timezone offset as created by `zoned.offset`.
```
this will fix the next few lines of formatting issues